_tools/release_docs/main.go

```diff
@@ -36,6 +36,7 @@ import (
 var (
 	branch       = flag.String("branch", "", "The git branch from which to pull docs. (e.g. release-1.0, master).")
 	outputDir    = flag.String("output-dir", "", "The directory in which to save results.")
+	version      = flag.String("version", "", "The release version. We assume this version will be relected in the generated pages' URL, e.g., we assume for release v1.0, its webpages will be hosted at k8s.io/v1.0.")
```

Comment:
It might also be good to mention that it is a required flag. (Same for output-dir)